core/controllers/admin_test.py

```diff
@@ -38,26 +38,28 @@ def setUp(self):
     def test_admin_page_rights(self):
         """Test access rights to the admin page."""
 
-        response = self.testapp.get('/admin')
+        response = self.get_html('/admin', expect_errors=True,
```

Comment:
Here and below, prefer breaking after '(' if contents of parens span multiple lines.

Reply:
done